src/components/RandomRecipePage.tsx

```diff
@@ -37,9 +27,8 @@ export function RandomRecipePage() {
       {recipe[0].title.length ? <div><h2>{recipe[0].title}</h2><img
         src={recipe[0].image}
         alt="Recipe" />
-        <br />
-        <a className="Recipe-link" target='_blank'
-          href={recipe[0].link}>See full recipe</a></div>
+        <p><a className="Recipe-link" rel="noreferrer" target='_blank'
```

Comment:
this is optional but normally classes should be lowercase kebab-case `recipe-link` or camelCase if you're writing jss.

but since that's opinion it wouldn't matter for a solo project like this - just a heads up when working with a team.